File: js/src/test/base/test.arraysConcat.js
// AUTO_TRANSPILE_ENABLED
import ccxt from '../../../ccxt.js';
import testSharedMethods from '../Exchange/base/test.sharedMethods.js';
function testArraysConcat() {
    const exchange = new ccxt.Exchange({
        'id': 'sampleexchange',
    });
    testSharedMethods.assertDeepEqual(exchange, undefined, 'testArraysConcat', exchange.arraysConcat([['b'], ['a', 'c']]), ['b', 'a', 'c']);
}
export default testArraysConcat;

File: julia/Ccxt/test/base/test.arraysConcat.jl
using Test
using Ccxt
function testArraysConcat()

    exchange = get(ccxt, Symbol("Exchange"), nothing)(Dict{Symbol, Any}(
        Symbol("id") => "sampleexchange"
    ));
    assertDeepEqual(testSharedMethods, exchange, nothing, "testArraysConcat", arraysConcat(exchange, [["b"], ["a", "c"]]), ["b", "a", "c"]);
end
